docs(readme): add installation commands (#3923)
ignite/cli: readme.md

## Quick start

> ### Installation

Reviewer: Alternative people on Linux can simply do `snap install ignite --classic`

Author: Thanks for the great suggestion @julienrbrt . The purpose of this PR was to resolve and issue raised about including the comments about creating the /usr/local/bin folder if we have an error. Do you want to add this? If so, please feel free to edit the readme.md, though please dont remove the comments I added as they work for both mac, linux and wsl.

Reviewer: Right, but we should strive to have an easier what to install ignite.
Now we got snap for Linux / WSL and on macOS we'll have homebrew soon™️.

If we keep the path instructions for h to use that install ignite without package manager, then we should make it collapse by default as imo it clutters the readme.

Author: Thanks @julienrbrt, I am fine with whatever you like and dont want it too cluttered either. Would you mind editing it to your liking? So I can close [#2249](https://github.com/ignite/cli/issues/2249)

Cheers

Author: @julienrbrt I am thinking it might be best to update the readme.md with snap and brew installs when both are available. I was asigned that issue and it required a PR, so prefer to just get it done and we can revisit later if need be, or perhaps open a PR to make the changes you want?

You can install Ignite with one simple command:
```
curl https://get.ignite.com/cli! | bash
```

> If Ignite is not automatically moved to your /usr/local/bin folder, you can do so with this command:

Reviewer 2: ```suggestion
If Ignite doesn't automatically move to your `/usr/local/bin` directory, use this command to do so:
```

```
sudo mv ignite /usr/local/bin
```

> In the event you get an error: you may need to create the /usr/local/bin folder and give yourself permission as follows:

Reviewer 2: ```suggestion
If you encounter an error, you might need to create the `/usr/local/bin` directory and set the necessary permissions as follows:
```

```
mkdir /usr/local/bin
sudo chown -R $(whoami) /usr/local/bin
```

> For more options on using and installing Ignite please see below:

Reviewer 2: ```suggestion
For more options on installing and using Ignite, please see the following:
```
